join_github_app/__init__.py

```diff
@@ -26,20 +26,17 @@ def create_app(github_service: GithubService, rate_limit: bool = True) -> Flask:
         format="%(asctime)s %(levelname)s in %(module)s: %(message)s",
     )
 
-    sentry_sdk.init(
-        dsn=os.environ.get("SENTRY_DSN_KEY"),
-        environment=os.environ.get("SENTRY_ENV"),
-        integrations=[FlaskIntegration()],
-        enable_tracing=True,
-        traces_sample_rate=0.1
-    )
+    if os.environ.get("SENTRY_DSN_KEY") and os.environ.get("SENTRY_ENV"):
```

Comment:
That's right! 😁 I thought I'd go for both values, since I believe the default environment is production 😬 so just in case people add the DSN key locally, but then forget the environment 🧠